src/dedicated_room/yuzu_room.cpp

```diff
@@ -75,6 +75,17 @@ static constexpr char BanListMagic[] = "YuzuRoom-BanList-1";
 
 static constexpr char token_delimiter{':'};
 
+static void PadToken(std::string& token) {
+    const auto remainder = token.size() % 3;
+    if (remainder == 0) {
+        return;
+    }
+
+    for (size_t i = 0; i < (3 - remainder); i++) {
+        token.push_back('=');
+    }
```

Comment:
```suggestion
    while (token.size() % 4 != 0) {
        token.push_back('=');
    }
```
where did 3 come from? the RFC says
```
                                                   Since all base64
   input is an integral number of octets, only the following cases can
   arise: (1) the final quantum of encoding input is an integral
   multiple of 24 bits; here, the final unit of encoded output will be
   an integral multiple of 4 characters with no "=" padding, (2) the
   final quantum of encoding input is exactly 8 bits; here, the final
   unit of encoded output will be two characters followed by two "="
   padding characters, or (3) the final quantum of encoding input is
   exactly 16 bits; here, the final unit of encoded output will be three
   characters followed by one "=" padding character.
```